tests/PresentationTests/Components/UI/SubmissionFormTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System.Security.Claims;
+using Application.Entities;
+using Application.Interfaces;
+using Application.Services;
+using Bunit;
+using Bunit.TestDoubles;
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.Forms;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.DependencyInjection;
+using Moq;
+using Presentation.Components.Pages;
+using Presentation.Components.UI;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace PresentationTests.Components.UI;
+public class SubmissionFormTests: TestContext
+{
+    private readonly ITestOutputHelper _testOutputHelper;
+    private const string TestJobName = "Test Job 2";
+
+    private readonly ApplicationUser _testUser =
+        new() { Id = "d45fc645-8176-4de2-83d5-5752bd0bb992", UserName = "devteam@de-simone.com" };
+
+    private readonly Project _testProject = new() { Id = 1, Number = "221013.22" };
+
+    public SubmissionFormTests(ITestOutputHelper testOutputHelper)
+    {
+        _testOutputHelper = testOutputHelper;
+        JSInterop.Mode = JSRuntimeMode.Loose;
+        Mock<UserManager<ApplicationUser>> mockUserManager =
+            new(Mock.Of<IUserStore<ApplicationUser>>(), null!, null!, null!, null!, null!, null!, null!, null!);
+        Services.AddSingleton(mockUserManager.Object);
+        var authContext = this.AddTestAuthorization();
+        authContext.SetAuthorized(_testUser.UserName!);
+        authContext.SetClaims(new Claim(ClaimTypes.NameIdentifier, _testUser.Id));
+        mockUserManager.Setup(service => service.FindByIdAsync(_testUser.Id)).Returns(Task.FromResult(_testUser)!);
+    }
+
+
+    [Fact]
+    public void test()
+    {
+        // Arrange
+
+        var jobServiceMock = new Mock<IJobService>();
+        var filePickerMock = new Mock<IFileSystemApiClient>();
+        var job = new Job
+        {
+            Id = 1,
+            Name = TestJobName,
+            Project = _testProject,
+            Creator = _testUser,
+            InputFile = "path/input.txt",
+            RuntimeConfigFile = "path/runtime.txt",
+            OutputFileDirectory = "path/output/",
+            SupplementalFilesDirectory = "path/supplemental/",
+        };
+
+        
+
+
+        jobServiceMock.Setup(service => service.CreateOrUpdateJob(job)).Returns(Task.CompletedTask);
+
+        var cxt = new TestContext();
+        var navManMock = new Mock<NavigationManager>();
+        cxt.Services.AddSingleton<NavigationManager>(navManMock.Object);
+        cxt.Services.AddSingleton<IJobService>(jobServiceMock.Object);
+        cxt.Services.AddSingleton<IFileSystemApiClient>(filePickerMock.Object);
+
+
+        var cut = cxt.RenderComponent<SubmissionForm>(parameters => parameters.Add(
```

Comment:
Same with ``RenderComponent`` method from base class.
```suggestion
        var cut = RenderComponent<SubmissionForm>(parameters => parameters.Add(
```